Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -182,7 +182,7 @@
           {
             "target": "project",
             "dataSourceName": "Projects",
-            "resultTemplate": "{ Value : \"{{id}}\", DisplayValue : \"{{{name}}}\" }"
+            "resultTemplate": "{ Value : \"{{id}}\", DisplayValue : \"{{{name}}} ({{id}})\" }"
```

Comment:
@GitHubSriramB Consider the following structure of projects and subprojects in TeamCity:

```
- ProjectA
  - CI
    - Build
  - Daily
    - Build
- ProjectB
  - CI
    - Build
  - Daily
    - Build
```

With the current implementation the build selection will look like this:

- Build
- Build
- Build
- Build

With the changes of this PR like this:

- Build (ProjectA_CI_Build)
- Build (ProjectA_Daily_Build)
- Build (ProjectB_CI_Build)
- Build (ProjectB_Daily_Build)

Beside the visual aspect, I also wasn't able to select a specific build definition with multiple names, since whatever entry I selected it always has chosen the same.